src/haxelib/client/Main.hx

```diff
@@ -188,6 +188,14 @@ class Main {
 	}
 
 
+	function checkUpdate() {
+		var latest:SemVer = try site.getLatestVersion("haxelib_client") catch (_:Dynamic) null;
+
+		if (latest != null && latest > VERSION) {
+			print('A new version (${latest}) of haxelib is available.\nDo "haxelib selfupdate" to get the latest version.\n');
```

Comment:
Don't forget to reword this to `haxelib update haxelib` as we want to deprecate `selfupdate`